lumibot/data_sources/ccxt_data.py

```diff
@@ -200,7 +214,7 @@ def get_barset_from_api(self, api, symbol, freq, limit=None, end=None):
 
     def _parse_source_symbol_bars(self, response, asset, quote=None, length=None):
         # Parse the dataframe returned from CCXT.
-        response["return"] = response["close"].pct_change()
+        response[:,"return"] = response["close"].pct_change()
```

Comment:
why was this change made? also, are you sure this works? please add a comment to explain

Reply:
Sorry about that. It's my mistake, I'll change it back.